lib/jsdom/utils.js

```diff
@@ -77,7 +77,7 @@ function addConstant(object, property, value) {
 }
 
 exports.mixin = (target, source) => {
-  const keys = Object.getOwnPropertyNames(source);
+  const keys = Reflect.ownKeys(source);
```

Comment:
Why change this?

Reply:
SVGListBase has symbol-named methods that also need to be mixed into child classes.